App.js

```diff
@@ -204,26 +204,7 @@ function ChatScreen({route, navigation}) {
           <Text style={{color: 'blue', fontSize: 18, marginBottom: 10}}>Link to another chat</Text>
         </Pressable>
         <Pressable
-          onPress={() => {
-            // This is fairly complex but seems to be the way we can "push" a new stack on and add the Settings screen in the routes
-            navigation.dispatch((state) => {
-              const newState = {...state};
-              const routes = [...newState.routes];
-              routes.push({
-                name: 'SettingsStack',
-                state: {
-                  index: 1,
-                  routes: [{name: 'Settings'}, {name: 'About'}]
-                }
-              });
-              const newestState = {
-                ...newState,
-                routes,
-                index: routes.length - 1,
-              };
-              return CommonActions.reset(newestState);
-            });
-          }}
+          onPress={() => navigation.push('SettingsStack', { screen: 'About' })}
```

Comment:
> I guess the downside is that every screen will need to define which screen comes before it instead of being able to pop() or goBack()?

Yeah that is the downside but I dont think it is that bad since it just makes sense that we will tell the navigation where to go when going up and goBack is easy

> And so, it has the same "back" behavior that we want here? And the "replace" for the "up" means we don't have to have the screen in the history to "pop" and can just swap the top screen with one that should come before it without pushing any history entries?

it has the same back behaviour afaik